XenModel/FriendlyNames.resx

```diff
@@ -1001,6 +1001,9 @@
 
 {1}</value>
   </data>
+  <data name="Message.body-gfs2_capacity" xml:space="preserve">
+    <value>Space utilization on SR '{0}' has exceeded 80%.</value>
+  </data>

```

Comment:
For my better understanding what is the distinction between the `body` and `name` messages?

Reply:
On the server side name is the key of the alert and body to message that appears to CLI users, e.g.:
```
uuid ( RO)         : xxxxx
         name ( RO): GFS2_CAPACITY
     priority ( RO): 1
        class ( RO): SR
     obj-uuid ( RO): xxxxx
    timestamp ( RO): 20200605T10:30:24Z
         body ( RO): SR utilisation above 80.0% for SR xxxx
```
In XenCenter we change this a bit and by name we refer to the short title of the alert when it first appears on the Alerts tab, while by body to the longer descritpion that appears once you expand the alert.